DevGuideExamples/FACE/two-topic/StockQuoter.idl

```diff
@@ -0,0 +1,25 @@
+#pragma FACE IDL_VERSION 3.1
+#include "orbsvcs/TimeBase.idl"
+module StockQuoter
+{
+  @topic  
+  struct Quote {
+    @key string ticker;
+    string exchange;
+    string full_name;
+    double value;
+    TimeBase::TimeT timestamp;
+  };
+
+  enum ExchangeEventType { TRADING_OPENED,
+                           TRADING_CLOSED,
+                           TRADING_SUSPENDED,
+                           TRADING_RESUMED };
+  @topic
+  struct ExchangeEvent {
+    @key string exchange;
+    ExchangeEventType event;
+    TimeBase::TimeT timestamp;
+  };
+  #pragma FACE ts_definitions Message
+};
```

Comment:
This file is missing an end of file newline.

Reply:
What I meant was that the red mark here in GitHub "Files changed" view is saying there's not a newline at the very end of the file. You should also be probably be getting a warning about it in your git client. I don't know what editor or IDE you're using, but [here's a general guide I found](https://thoughtbot.com/blog/no-newline-at-end-of-file).